go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0
-	github.com/ava-labs/avalanchego v1.9.11
+	github.com/ava-labs/avalanchego v1.10.0-fuji-rc.2
```

Comment:
Why rc.2 rather than the actual fuji release?

Reply:
I wasn't sure which of the rcs and -fuji were the final one, so checked a slack channel to see which one was used. Since these are the same, going to leave this as is since it only needs to update the `RPCChainVMProtocolVersion`.